aspnetcore/security/authentication/azure-ad-b2c.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Cloud authentication with Azure Active Directory B2C
+author: camsoper
+description: Shows how to set up Azure Active Directory B2C with ASP.NET Core
+keywords: ASP.NET Core,authentication,AAD,B2C,Azure Active Directory,Active Directory
+ms.author: casoper
+manager: wpickett
+ms.date: 01/12/2018
+ms.topic: article
+ms.technology: aspnet
+ms.prod: asp.net-core
+uid: security/authentication/azure-ad-b2c
+---
+# Cloud authentication with Azure Active Directory B2C
+
+By [Cam Soper](https://twitter.com/camsoper)
+
+[Azure Active Directory B2C](https://docs.microsoft.com/azure/active-directory-b2c/active-directory-b2c-overview) (AAD B2C) is a cloud identity management solution for your web and mobile applications.  The service provides global, scalable, cloud-based authentication to web applications and APIs using individual accounts, social network accounts, and federated enterprise accounts for applications hosted in the cloud and on-premises.  Additionally, AAD B2C can provide multifactor authentication with very little configuration.
+
+This page demonstrates using Azure Active Directory B2C with ASP.NET Core.
+
+## Prerequisites
+
+The following are required for this walkthrough:
+
+* A [Microsoft Azure subscription](https://azure.microsoft.com/free/?ref=microsoft.com&utm_source=microsoft.com&utm_medium=docs&utm_campaign=visualstudio).  
+* [Visual Studio 2017](https://www.visualstudio.com/vs/) (any edition).
```

Comment:
I see this kind of link being used to download on VS docs: https://aka.ms/vsdownload?utm_source=mscom&utm_campaign=msdocs. Not sure if they use that to track how much traffic docs send them. @TerryGLee do you know?

Reply:
Yes, thanks for asking. :) This is the tracking URL that the product team would like us to use, as it helps them determine where the downloads are coming from, and this one points to the source as from Docs.Microsoft.com.

Reply:
I opened an issue to inquire further :point_right: https://github.com/aspnet/Docs/issues/5204